cordova-lib/src/hooks/HooksRunner.js

```diff
@@ -80,6 +96,15 @@ HooksRunner.prototype.prepareOptions = function(opts) {
     return opts;
 };
 
+function HooksRunner(projectRoot) {
```

Comment:
Could you please provide the 'why' we are moving to have both CordovaHooksRunner and DefaultHooksRunner ? So that other maintainers are not confused as to the reason behind this shift.